### ISSUE TITLE
Visual Studio Marketplace

### ISSUE THREAD

**Issue by NN---** [HexWrench/VisualHG2015#3]
Update extension.

There is an old version: https://marketplace.visualstudio.com/items?itemName=JayBeaversWork.VisualHG
Without any icon.

It is not clear which VisualHG user should pick.
Maybe a different name is needed since there are already several clones like: https://bitbucket.org/lmn/visualhg2/overview :)

Perhaps something like: TortoiseHg Visual Studio Integration.

**Comment by HexWrench** [comment]
There is an official Tortoise HG plugin (which is not free though). 
In any case, I won't be maintaining this an longer since we move all of our repos over to Git.

**Comment by NN---** [comment]
What is the official plugin?
I don't know about anything.

Ok,  can I clone this repo then and maintain vsix in gallery on my own?

**Comment by HexWrench** [comment]
The TortoiseHG homepage links to this: [HgVS](https://hgvs.multicorewareinc.com/download/).
But feel free to clone my repo and maintain it (and put it in the gallery). I myself cloned the original VisualHG repo because it wasn't maintained anymore.

**Comment by NN---** [comment]
So first it is free now.
Second I really like integration with TortoiseHG.
Also this one didn't update for long time "Last updated
9.11.2015, 06:50:34"

**Comment by NN---** [comment]
Btw why you abandon mercurial and go to Git?

**Comment by NN---** [comment]
I agree about tooling. I understood it was a licensing problem since there is permissive libgit2 and nothing compared in mercurial.

Do you know maybe anything compared to query language of mercurial in Git? This one is very missing when I work with Git repositories.

**Comment by NN---** [comment]
@HexWrench Is there a compiled vsix I can download ?
Thanks.
